draft-ietf-scitt-architecture.md

```diff
@@ -375,7 +374,7 @@ This section describes at a high level, the three main roles and associated proc
 ## Transparency Service
 
 Transparency Services MUST feature an Append-only Log.
-The Append-only Log is the verifiable data structure that registers Signed Statements and supports the production of Receipts.
+The Append-only Log is a verifiable data structure that accumulates all Signed Statements registered by the Transparency Service and that supports the production of Receipts.
```

Comment:
Proposal to revert this change, as "accumulates", as we've moved away from the Transparency Service recording Signed Statement contgents, as that may introduce PII or other redaction scenarios that would be impossible. Rather the Transparency Service registers signed statements, producing receipts.
```suggestion
The Append-only Log is the verifiable data structure that records Signed Statements and supports the production of Receipts.
```

Reply:
I agree with @SteveLasker assessment and I would prefer the earlier `old` version of the text.

In fact I would re-phrase it to:
`An Append-only Log is the verifiable data structure that registers Signed Statements and produces Receipts.`